lti_consumer/lti_consumer.py

```diff
@@ -860,3 +860,15 @@ def _get_modal_position_offset(self, viewport_percentage):
             float: The css position offset to apply to the modal window
         """
         return (100 - viewport_percentage) / 2
+
+    def get_outcome_service_url(self, service_name="grade_handler"):
+        """
+        This function is called by get_course_lti_endpoints when using LTI result service to
+        discover the LTI result endpoints. It is similar to outcome_service_url property but
+        with a mapping for backward compatibility.
+        """
+        mapping = {
+            'grade_handler': 'outcome_service_handler',
+            'lti_2_0_result_rest_handler': 'result_service_handler'
+        }
+        return self.runtime.handler_url(self, mapping[service_name], thirdparty=True).rstrip('/?')
```

Comment:
Sorry I forgot about this.  Here is untested code to explain what I meant.  I still feel new to Python, so not sure if there are any issues with this.
```
def get_outcome_service_url(self, service_name="grade_handler"):
        """
        This function is called by get_course_lti_endpoints when using LTI result service to
        discover the LTI result endpoints.
        """
        mapping = {
            'grade_handler': outcome_service_url,
            'lti_2_0_result_rest_handler': result_service_url,  # new function needed
        }
        return mapping[service_name]()
```
I removed the docstring comment about `outcome_service_url`, because this function now just calls the appropriate property, which has its own comments.

Does this make my original comment any more clear?  Does this seem reasonable @xcompass ? @douglashall?